### PR TITLE
Add staging repo for etcd images

### PR DESCRIPTION
**Pull request by jingyih**
Will host images for https://github.com/kubernetes/kubernetes/tree/master/cluster/images/etcd

### CODE REVIEW

File: k8s.gcr.io/images/k8s-staging-etcd/OWNERS [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```
Comment (jingyih) [inline k8s.gcr.io/images/k8s-staging-etcd/OWNERS:1]: I copied owners from https://github.com/kubernetes/kubernetes/blob/master/cluster/images/etcd/OWNERS
and added myself. Should I add myself to the original OWNERS file first?

Comment (spiffxp) [reply k8s.gcr.io/images/k8s-staging-etcd/OWNERS:1]: I would like to see you added to that file, yes, but I don't consider it a blocker for this

**Comment by spiffxp** [comment]
```
groups (master)$ make run -- --confirm
git-crypt unlock
GO111MODULE=on go run reconcile.go --confirm
2020/04/03 10:20:58  ======================= Updates =========================
2020/04/03 10:22:47 Trying to create group: "k8s-infra-staging-etcd@kubernetes.io"
2020/04/03 10:22:48 > Successfully created group k8s-infra-staging-etcd@kubernetes.io
2020/04/03 10:22:49 > Successfully updated group settings for "k8s-infra-staging-etcd@kubernetes.io" to allow external members and other security settings
2020/04/03 10:22:50 Added jpbetz@google.com to "k8s-infra-staging-etcd@kubernetes.io" as a OWNER
2020/04/03 10:22:51 Added wenjiazhang@google.com to "k8s-infra-staging-etcd@kubernetes.io" as a OWNER
2020/04/03 10:22:52 Added jingyih@google.com to "k8s-infra-staging-etcd@kubernetes.io" as a OWNER
git-crypt lock
```
```
k8s.io (master)$  ./infra/gcp/ensure-staging-storage.sh etcd
Configuring staging: etcd
Ensuring project exists: k8s-staging-etcd
Create in progress for [https://cloudresourcemanager.googleapis.com/v1/projects/k8s-staging-etcd].
Waiting for [operations/cp.5610427005476136120] to finish...done.
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-staging-etcd/billingInfo
projectId: k8s-staging-etcd
Empowering k8s-infra-staging-etcd@kubernetes.io as project viewers
Updated IAM policy for project [k8s-staging-etcd].
bindings:
- members:
  - user:spiffxp@google.com
  role: roles/owner
- members:
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/viewer
etag: BwWiZjZr8vM=
version: 1
Enabling the container registry API
Operation "operations/acf.e0a20f7f-035d-474b-9a4d-808bcde23b13" finished successfully.
Ensuring the registry exists and is readable
Using default tag: latest
latest: Pulling from pause
a3ed95caeb02: Pull complete
4964c72cd024: Pull complete
Digest: sha256:a78c2d6208eff9b672de43f880093100050983047b7b0afe0217d3656e1b0d5f
Status: Downloaded newer image for k8s.gcr.io/pause:latest
k8s.gcr.io/pause:latest
The push refers to repository [gcr.io/k8s-staging-etcd/ceci-nest-pas-une-image]
5f70bf18a086: Layer already exists
e16a89738269: Pushed
latest: digest: sha256:ec3ca3ee90e4dafde96c83232b30f17b5e8992ff35479d0661b8f4ff2f21bf74 size: 938
Digests:
- gcr.io/k8s-staging-etcd/ceci-nest-pas-une-image@sha256:ec3ca3ee90e4dafde96c83232b30f17b5e8992ff35479d0661b8f4ff2f21bf74
  Associated tags:
 - latest
Tags:
- gcr.io/k8s-staging-etcd/ceci-nest-pas-une-image:latest
Deleted [gcr.io/k8s-staging-etcd/ceci-nest-pas-une-image:latest].
Deleted [gcr.io/k8s-staging-etcd/ceci-nest-pas-une-image@sha256:ec3ca3ee90e4dafde96c83232b30f17b5e8992ff35479d0661b8f4ff2f21bf74].
Enabling Bucket Policy Only for gs://artifacts.k8s-staging-etcd.appspot.com...
Empowering GCR admins
Updated IAM policy for project [k8s-staging-etcd].
bindings:
- members:
  - serviceAccount:service-329483391043@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:spiffxp@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/viewer
etag: BwWiZjpBK2Q=
version: 1
Empowering k8s-infra-staging-etcd@kubernetes.io to GCR
Enabling the GCS API
Operation "operations/acf.f7e0f2c3-d18c-4c1a-941b-f4004b9ceb77" finished successfully.
Configuring bucket: gs://k8s-staging-etcd
Ensuring the bucket exists and is world readable
Creating gs://k8s-staging-etcd/...
Enabling Bucket Policy Only for gs://k8s-staging-etcd...
Ensuring the bucket has auto-deletion of 60 days
Setting lifecycle configuration on gs://k8s-staging-etcd/...
Empowering GCS admins
Updated IAM policy for project [k8s-staging-etcd].
bindings:
- members:
  - serviceAccount:service-329483391043@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:spiffxp@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/viewer
etag: BwWiZjv607o=
version: 1
Empowering k8s-infra-staging-etcd@kubernetes.io to GCS
Configuring bucket: gs://k8s-staging-etcd-gcb
Ensuring the bucket exists and is world readable
Creating gs://k8s-staging-etcd-gcb/...
Enabling Bucket Policy Only for gs://k8s-staging-etcd-gcb...
Ensuring the bucket has auto-deletion of 60 days
Setting lifecycle configuration on gs://k8s-staging-etcd-gcb/...
Empowering GCS admins
Updated IAM policy for project [k8s-staging-etcd].
bindings:
- members:
  - serviceAccount:service-329483391043@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:spiffxp@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/viewer
etag: BwWiZj02JEc=
version: 1
Empowering k8s-infra-staging-etcd@kubernetes.io to GCS
Enabling the KMS API
Operation "operations/acf.3c40cef6-3ee9-404a-8037-2740d68315a4" finished successfully.
Enabling the GCB API
Operation "operations/acf.b2d171bf-e453-4469-8a61-9f11cc5dd426" finished successfully.
Empowering k8s-infra-staging-etcd@kubernetes.io as GCB editors
Updated IAM policy for project [k8s-staging-etcd].
bindings:
- members:
  - serviceAccount:329483391043@cloudbuild.gserviceaccount.com
  role: roles/cloudbuild.builds.builder
- members:
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/cloudbuild.builds.editor
- members:
  - serviceAccount:service-329483391043@gcp-sa-cloudbuild.iam.gserviceaccount.com
  role: roles/cloudbuild.serviceAgent
- members:
  - serviceAccount:service-329483391043@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:spiffxp@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/viewer
etag: BwWiZj75uvY=
version: 1
Updated IAM policy for project [k8s-staging-etcd].
bindings:
- members:
  - serviceAccount:329483391043@cloudbuild.gserviceaccount.com
  role: roles/cloudbuild.builds.builder
- members:
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/cloudbuild.builds.editor
- members:
  - serviceAccount:service-329483391043@gcp-sa-cloudbuild.iam.gserviceaccount.com
  role: roles/cloudbuild.serviceAgent
- members:
  - serviceAccount:service-329483391043@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:spiffxp@google.com
  role: roles/owner
- members:
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/serviceusage.serviceUsageConsumer
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/viewer
etag: BwWiZj8ZttI=
version: 1
Empowering Prow
Updated IAM policy for project [k8s-staging-etcd].
bindings:
- members:
  - serviceAccount:329483391043@cloudbuild.gserviceaccount.com
  - serviceAccount:deployer@k8s-prow.iam.gserviceaccount.com
  role: roles/cloudbuild.builds.builder
- members:
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/cloudbuild.builds.editor
- members:
  - serviceAccount:service-329483391043@gcp-sa-cloudbuild.iam.gserviceaccount.com
  role: roles/cloudbuild.serviceAgent
- members:
  - serviceAccount:service-329483391043@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:spiffxp@google.com
  role: roles/owner
- members:
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/serviceusage.serviceUsageConsumer
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/viewer
etag: BwWiZj85_pI=
version: 1
Done
Configuring special cases for Release Managers on: kubernetes
Empowering k8s-infra-release-viewers@kubernetes.io as project viewers
Updated IAM policy for project [k8s-staging-etcd].
bindings:
- members:
  - serviceAccount:329483391043@cloudbuild.gserviceaccount.com
  - serviceAccount:deployer@k8s-prow.iam.gserviceaccount.com
  role: roles/cloudbuild.builds.builder
- members:
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/cloudbuild.builds.editor
- members:
  - serviceAccount:service-329483391043@gcp-sa-cloudbuild.iam.gserviceaccount.com
  role: roles/cloudbuild.serviceAgent
- members:
  - serviceAccount:service-329483391043@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:spiffxp@google.com
  role: roles/owner
- members:
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/serviceusage.serviceUsageConsumer
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-release-viewers@kubernetes.io
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/viewer
etag: BwWiZj-iYZY=
version: 1
Configuring special cases for Release Managers on: release-test
Empowering k8s-infra-release-viewers@kubernetes.io as project viewers
Updated IAM policy for project [k8s-staging-etcd].
bindings:
- members:
  - serviceAccount:329483391043@cloudbuild.gserviceaccount.com
  - serviceAccount:deployer@k8s-prow.iam.gserviceaccount.com
  role: roles/cloudbuild.builds.builder
- members:
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/cloudbuild.builds.editor
- members:
  - serviceAccount:service-329483391043@gcp-sa-cloudbuild.iam.gserviceaccount.com
  role: roles/cloudbuild.serviceAgent
- members:
  - serviceAccount:service-329483391043@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:spiffxp@google.com
  role: roles/owner
- members:
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/serviceusage.serviceUsageConsumer
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-release-viewers@kubernetes.io
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/viewer
etag: BwWiZj_AF6E=
version: 1
Configuring special cases for Release Managers on: releng
Empowering k8s-infra-release-viewers@kubernetes.io as project viewers
Updated IAM policy for project [k8s-staging-etcd].
bindings:
- members:
  - serviceAccount:329483391043@cloudbuild.gserviceaccount.com
  - serviceAccount:deployer@k8s-prow.iam.gserviceaccount.com
  role: roles/cloudbuild.builds.builder
- members:
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/cloudbuild.builds.editor
- members:
  - serviceAccount:service-329483391043@gcp-sa-cloudbuild.iam.gserviceaccount.com
  role: roles/cloudbuild.serviceAgent
- members:
  - serviceAccount:service-329483391043@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:spiffxp@google.com
  role: roles/owner
- members:
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/serviceusage.serviceUsageConsumer
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-release-viewers@kubernetes.io
  - group:k8s-infra-staging-etcd@kubernetes.io
  role: roles/viewer
etag: BwWiZj_gGDI=
version: 1
```

**Pull request merged**